src/pages/Login/LoginPage.tsx

```diff
@@ -259,7 +260,7 @@ export class LoginPage extends React.Component<LoginProps, LoginState> {
 
     return (
       <LoginNext
-        footerListVariants="inline"
+        footerListVariants={ListVariant.inline}
         brandImgSrc={kialiLogo}
```

Comment:
On the other hand they create the variable for this ^^ I can't understand how they do to  create a new one and break others